gpx.cc

```diff
@@ -972,7 +972,7 @@ gpx_end(const QString& el)
      * last date we saw in this log.
      */
   case tt_cache_log_type:
-    if ((cdatastr.compare("Found it") == 0) &&
+    if ((cdatastr.compare(QString("Found it")) == 0) &&
```

Comment:
Most of these look like mechanical conversions from C. Would they be more readable with the cdatastr == QStringLiteral() form since QString has overloaded operator == ?